integration-tests/bats/remotes.bats

```diff
@@ -1001,7 +1090,12 @@ SQL
     [ "$status" -eq 0 ]
 }
 
-@test "remotes: DOLT_CHECKOUT to checkout to a remote branch." {
+@test "remotes: DOLT_CHECKOUT new branch without '-b' to checkout to set upstream to a remote branch." {
```

Comment:
some of these test names could be a little more descriptive/clear. 

For example, from the names, I have a hard time understanding the difference in intent between this test and the one on line 123, named: "remotes: 'dolt checkout remote_branch_name' sets upstream".